daprdocs/content/en/js-sdk-docs/js-server/_index.md

```diff
@@ -66,6 +66,49 @@ npm run start:dapr-http
 
 > ℹ️ **Note:** The `app-port` is required here, as this is where our server will need to bind to. Dapr will check for the application to bind to this port, before finishing start-up.
 
+### Using HTTP (bring your own express webserver)
+
+By default, for sidecar to application communication we bring a webserver that is configured to listen to the Sidecar calls as defined in the Dapr API specification. However, in some scenarios (e.g., when you are building a REST API backend and want to integrate Dapr directly in it) you want to bring your own webserver.
+
+As the Dapr JS SDK team, we do support this scenario, but currently this is only available for [`express`](https://www.npmjs.com/package/express).
+
+> 💡 Note: when bringing your own webserver, we will introduce certain overrides that might happen (e.g., configuring the body size, adding certain routes, ...). We avoid collisions in routes by having our own uniques ones, but cannot guarantee collissions.
```

Comment:
```suggestion
> 💡 Note: when using a custom web-server, the SDK will configure server properties like max body size, and add new routes to it. The routes are unique on their own to avoid any collisions with your application, but it's not guaranteed to not collide.
```

1. Should we configure the body size if user brings their own server? That option should be ignored in this case IMO.
2. Should we customize Dapr routes with a specific prefix? for example - `/dapr-server-<route>`, to ensure zero collisions?

Reply:
Docs adaptation: OK

1. I think we should, we should still have tests passing and we require certain middleware for certain body types. Else we depend on the user to handle cloudevents for example
2. For now I would keep AS-IS, let's see if users require it first seeing the high impact of this

Reply:
I was just wondering if configuring max body limit on user's express instance will be intrusive, but I see your point. We should wait for some feedback on this feature and take a call I guess!